From: Chris Diamand <chris@diamand.org>
Date: Tue, 29 May 2012 17:34:15 +0200
Subject: [PATCH] Changed call to subprocess.check_output which isn't
 compatible with Python 2.6

subprocess.check_output is not present in Python 2.6. Since the return value
is not being used anyway, replace it with subprocess.call(), which is.

Signed-off-by: Chris Diamand <chris@diamand.org>
Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>
---
 opkg-make-index | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/opkg-make-index b/opkg-make-index
index 22393e8..1c3a8e1 100755
--- a/opkg-make-index
+++ b/opkg-make-index
@@ -186,7 +186,7 @@ if packages_filename:
      gzip_filename = ("%s.gz" % packages_filename)
      tmp_gzip_filename = ("%s.%d" % (gzip_filename, os.getpid()))
      gzip_cmd = "gzip -9c < %s > %s" % (tmp_packages_filename, tmp_gzip_filename)
-     rc = subprocess.check_output(gzip_cmd, shell=True)
+     subprocess.call(gzip_cmd, shell=True)
      os.rename(tmp_packages_filename, packages_filename)
      os.rename(tmp_gzip_filename, gzip_filename)
 
